pre-docs/bolt_configuration_options.md

```diff
@@ -52,6 +55,8 @@ ssh:
 
 `tmpdir`: The directory to upload and execute temporary files on the target.
 
+`interpreters`: A map of extension name to absolute path of an executable. This allows a user to override the shebang defined in a task executable. 
```

Comment:
Waiting for review before I paste this in the other relevant transports. Also will need to add a note that the `local` transport will default to bolt's ruby.